docs/source/conf.py

```diff
@@ -39,26 +40,74 @@
 
 
 # Get key metadata from google sheet
-doc_id = "1tHXC8ZH6_v_SaR1tRffSZCphZL6Y6ktDYp6y4ysE0gg"
+doc_id = "1z1bh55BFTwp7u-069PD8NF6r_ZmpCQwr7i78W6RBY_g"
+sheet_id = "0"
+sheet_url = (
+    f"https://docs.google.com/spreadsheets/d/{doc_id}/export?format=csv&gid={sheet_id}"
+)
+df_all = pd.read_csv(sheet_url, header=3)
 
 
-def download_metadata_keys(sheet_id: str, csv_file_name: str) -> None:
-    sheet_id = sheet_id
-    sheet_url = f"https://docs.google.com/spreadsheets/d/{doc_id}/export?format=csv&gid={sheet_id}"
-    df = pd.read_csv(sheet_url)
-    df = df.replace("\*", "", regex=True)
-    df.to_csv(csv_file_name, index=False)
+df_list = np.split(df_all, df_all[df_all.isnull().all(1)].index)
+
+# Remove all CSV from the folder
+path_csv_folder = os.path.join("C:\\projects\\disdrodb-fork\\docs\\source", "csv")
+filelist = [f for f in os.listdir(path_csv_folder) if f.endswith(".csv")]
+for f in filelist:
+    os.remove(os.path.join(path_csv_folder, f))
+
+
+# create rst text
+metadata_keys_text = """
+=========================
+Metadata keys
+=========================
+
 
+The following tables are based on `this google sheet <https://docs.google.com/spreadsheets/d/1z1bh55BFTwp7u-069PD8NF6r_ZmpCQwr7i78W6RBY_g>`__. 

```

Comment:
Here we provide a link that enable edit the metadata.
You should eventually here use a link that enable just a view mode @regislon  ;)

Reply:
yes, but only the owner of the google sheet can do that @ghiggi.

Reply:
Ok. I can't generate two types of links (edit and view mode). 
Maybe we could directly add the full csv in the docs? Or already the separated csv? @regislon ?

Reply:
the csv is generated dynamically while builing the doc